sync.sh

```diff
@@ -0,0 +1,17 @@
+
+APP_NAME="tdp-backend-sandbox"
+APP_GUID=$(cf curl /v3/apps/$(cf app $APP_NAME --guid)/processes | jq --raw-output '.resources | .[] | select(.type == "web").guid')
+
+echo "APP NAME:$APP_NAME"
+echo "APP GUID:$APP_GUID"
+
+cat <<EOF > ~/.ssh/config
+Host $APP_NAME
+    HostName ssh.fr.cloud.gov
+    User cf:$APP_GUID/0
+    Port 2222
+EOF
+
+cat ~/.ssh/config
```

Comment:
This whole block is basically what my script `update-ssh-config.sh` does. 

1. Could you move `sync.sh` to the scripts folder?
2. Could you incorporate my script `update-ssh-config.sh`?
3. Retest sync to ensure it works
4. Update script folder README for `sync.sh`